### PR TITLE
Disney characters

### PR DESCRIPTION
**Pull request by haliljon**
## In this feature, we:

-  Added a home page showing a list of items that you can "like."
-  Added a Popup Window once the user clicks on the comment button
-  Added an external website when the user clicks on the open giveaway button
-  Added a footer with Copyright mark
-  Added functionalities in JS using ES6 modules

image

## Video presentation link: 
https://drive.google.com/file/d/119ysGKwqPH1ndc5VFvvo-9RoZqOIlgk_/view

### CODE REVIEW

File: src/test/countComments.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <li>5</li>
+       </ul>`;
+  });
+
+  test('it should print 3', () => {
+    const ul = document.querySelector('.ul-comment');
+    const result = countComments(ul);
+    expect(result).toBe(3);
+  });
+});
```
Comment (ShadyShawkat) [inline src/test/countComments.test.js:21]: - [ ] Good job implementing a test for the counter function. Please make sure that you have tests for all edge cases like when there are 0 items on the page (counter function should return 0).

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/countComments.test.js (hunk 1 above)
Comment (annaeliana7) [reply src/test/countComments.test.js]: @ShadyShawkat, 
I have completed the implementation of the test and have pushed the update. Please review and approve. Thank you.

**Review by KDlamini — APPROVED** [review]
### Hi @haliljon, @annaeliana7 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Professional pull request
✔️Project uses ES6 and Webpack correctly
✔️ Great UI implementation and API integration
✔️ Correct use of GitHub flow
✔️ No linter errors

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
